ClassTrigger.trigger

```diff
@@ -0,0 +1,26 @@
+trigger ClassTrigger on Class__c (before insert, before delete, before update) {
```

Comment:
PR should have appropriate tittle and description

Reply:
Ok